app/models/deploy_service.rb

```diff
@@ -108,4 +109,10 @@ def update_github_deployment_status(deploy)
       GithubDeployment.new(deploy).update_github_deployment_status(@deployment)
     end
   end
+
+  def send_sse_deploy_update(type, deploy)
+    return unless deploy.persisted?
+    rebased_deploy = JSON.parse(DeploySerializer.new(deploy).to_json)["deploy"]
```

Comment:
maybe expose a method that shows the hash before it is jsonified like .present / .to_h instead of generating and parsing :)